service/service_test.go

```diff
@@ -90,11 +91,19 @@ func TestRun(t *testing.T) {
 			return kafkaProducerMock, nil
 		}
 
+		funcDoGetHealthClientOk := func(name string, url string) *health.Client {
```

Comment:
Just wonder if there should be an additional test for the health client not being OK.

Reply:
The Health Client constructor doesn't return error - basically it creates a new HTTP Client without retries in order to request health states of other services. I don't know what failure test case could we create?

Reply:
OK, my misunderstanding from the ones I've been doing previously. Ignore this comment :-)